app/jobs/concerns/hyku_addons/portable_bulkrax_entry_behavior.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+# rubocop:disable Rails/Output
```

Comment:
This disable statement should be removed, along with the puts statement before merge